src/qt/veil.cpp

```diff
@@ -146,15 +146,19 @@ static void initTranslations(QTranslator &qtTranslatorBase, QTranslator &qtTrans
 /* qDebug() message handler --> debug.log */
 void DebugMessageHandler(QtMsgType type, const QMessageLogContext& context, const QString &msg)
 {
-#ifndef QT_MESSAGELOGCONTEXT
-   Q_UNUSED(context);
+#if 0
+
+    // Please don't enable this publicly. It may leak information, and we've been bit by a nasty crash where the context is not available in non-debug builds.
+
+LogPrintf("GUI: %s:%d %s\n", context.file, context.line, msg.toStdString());
+
+#else
+    Q_UNUSED(context);
```

Comment:
Just comment it out simply; so that it doesn't change the functionality of the #ifdef that _is_ required.

What I'm asking for is simple.

```
// #ifdef QT_MESSAGELOGCONTEXT
// Do not use this except in debugging environments as some OSs don't handle it properly
//QT_MESSAGELOGCONTEXT needs to be added to the DEFS line of the Makefile
//    LogPrintf("GUI: %s:%d %s\n", context.file, context.line, msg.toStdString());
// #else
```